### DISCUSSION TITLE
Does Spotube "tells" Spotify which tracks I listening to?

### DISCUSSION THREAD

**Discussion by IkorJefocur** [KRTirtho/spotube#844] [Q&A] [answered]
On Spotify, track listening history is used for generate personal playlists, like daily mixes, etc.. So I wondering, does this feature still working when I'm using Spotube? I know that someone can call this tracking and prefer to avoid this instead for privacy reasons, but I'd prefer to keep this feature.

**Comment by KRTirtho** [comment, marked as answer]
Nope unfortunately this is not done by Spotube both for privacy reasons and limitation of the Spotify API
